web/src/modules/common/components/Tooltip.js

```diff
@@ -38,64 +38,79 @@ type FlowType = 'left' | 'right' | 'up' | 'down'
 const pseudosMixin = (flow: FlowType, supportsLogicalProperties: boolean) => css`
   /* CSS Triangle: https://css-tricks.com/snippets/css/css-triangle/ */
   ::before {
-    ${flow === 'up' && `
+    ${flow === 'up' &&
+    `
       bottom: 100%;
       border-bottom-width: 0;
       border-top-color: #333;
     `}
-    ${flow === 'down' && `
+    ${flow === 'down' &&
+    `
       top: 100%;
       border-top-width: 0;
       border-bottom-color: #333;
     `}
-    ${flow === 'left' && `
+    ${flow === 'left' &&
+    `
       ${toLogicalProperty('border-right-width', supportsLogicalProperties)}: 0;
       ${toLogicalProperty('border-left-color', supportsLogicalProperties)}: #333;
       ${toLogicalProperty('left', supportsLogicalProperties)}: -5px;
     `}
-    ${flow === 'right' && `
+    ${flow === 'right' &&
+    `
       ${toLogicalProperty('border-left-width', supportsLogicalProperties)}: 0;
       ${toLogicalProperty('border-right-color', supportsLogicalProperties)}: #333;
       ${toLogicalProperty('right', supportsLogicalProperties)}: -5px;
     `}
   }
-  
+
   ::after {
-    ${flow === 'up' && `
+    ${flow === 'up' &&
+    `
       bottom: calc(99% + 5px);
     `}
```

Comment:
those expressions actually seem uneccessary lengthy. But didn't find an option to not break quotes, so I think it is fine.